### PR TITLE
Testing comments only

### PR DESCRIPTION
**Pull request by vivektailang**
Testing comments only

### CODE REVIEW

File: src/main/java/ExcelReaderCode.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public static void main(String[] args) throws IOException {
         //This is my changes
         int cols = sh.getRow(0).getLastCellNum();
         int cols2 = sh.getRow(0).getLastCellNum();
-        //
+        //Testing comments
```
Comment (gtgauravtelang) [inline src/main/java/ExcelReaderCode.java:32]: Please remove it.

**Review by gtgauravtelang — CHANGES_REQUESTED** [review]
kindly remove your comment as instructed.

File: src/main/java/ExcelReaderCode.java (hunk 1 above)
Comment (vivektailang) [reply src/main/java/ExcelReaderCode.java:32]: not mandatory to remove

Comment (gtgauravtelang) [reply src/main/java/ExcelReaderCode.java:32]: Ok

**Review by gtgauravtelang — APPROVED** [review]
Resolved.

**Pull request merged**